modules/administration_manual/pages/configuration/files/encryption/enable-encryption.adoc

```diff
@@ -0,0 +1,55 @@
+[[enable-the-encryption-app]]
```

Comment:
hmmm,
we have (above)
``enable-encryption-app.adoc``
and this file 
``enable-encryption.adoc`` which seem sto be contentwise a dupe of the above 
?

Reply:
You're right. I'm not sure how it happened though.

Reply:
Sorted out!